source/stylesheets/refills/_animate-info.scss

```diff
@@ -0,0 +1,15 @@
+.animate-info {
+  line-height: 1.5;
+  font-size: 0.9em;
+  color: #909090;
+  margin: auto;
+  max-width: 22em;
+
+  a {
+    color: $refills-anchor-color;
+    text-decoration: none;
+  }
+
+  margin-top: 3em;
```

Comment:
`margin-top` is just overwriting `margin` defined above; these should be combined. Maybe use Bourbon’s helper (`@include margin(3em auto null);`)?